File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.cpp
class Solution {
public:
    vector<int> productExceptSelf(vector<int>& nums) {
        vector<int> res;
        for (int i = 0; i < nums.size(); i++) {
            int curres = 1;
            for (int j = 0; j < nums.size(); j++) {
                if (i == j) continue;
                curres = curres * nums[j];
            }
            res.push_back(curres);
        }
        return res;
    }
};

File: Data Structures & Algorithms/merge-two-sorted-linked-lists/submission-0.py
# Definition for singly-linked list.
# class ListNode:
#     def __init__(self, val=0, next=None):
#         self.val = val
#         self.next = next

class Solution:
    def mergeTwoLists(self, list1: Optional[ListNode], list2: Optional[ListNode]) -> Optional[ListNode]:
        #Brute Force
        nums = []
        while list1:
            nums.append(list1.val)
            list1 = list1.next
        while list2:
            nums.append(list2.val)
            list2 = list2.next
        
        nums.sort()

        res = ListNode()
        curr = res

        for num in nums:
            curr.next = ListNode(num)
            curr = curr.next

        return res.next